### PR TITLE
Some enhancements for Dottydoc and ScalaDays

### PR DESCRIPTION
**Pull request by felixmulder**

### CODE REVIEW

File: doc-tool/src/dotty/tools/dottydoc/model/internal.scala [hunk 1]
```diff
@@ -86,13 +86,15 @@ object internal {
...
-    modifiers: List[String],
+    private val mods: List[String],
     path: List[String],
     superTypes: List[MaterializableLink] = Nil,
     var comment: Option[Comment] = None,
     var companionPath: List[String] = Nil,
     var parent: Entity = NonEntity
-  ) extends Object
+  ) extends Object {
+    def modifiers: List[String] = mods.filterNot(_ == "private")
```
Comment (DarkDimius) [inline doc-tool/src/dotty/tools/dottydoc/model/internal.scala]: should it be "final"?

Comment (felixmulder) [inline doc-tool/src/dotty/tools/dottydoc/model/internal.scala]: waps, it should indeed!

File: compiler/src/dotty/tools/dotc/config/ScalaSettings.scala [hunk 2]
```diff
@@ -123,5 +123,12 @@ class ScalaSettings extends Settings.SettingGroup {
     sys.props("user.dir").split(File.separatorChar).last
   )
 
+  val projectVersion = StringSetting (
+    "-project-version",
+    "project version",
+    "The current version of your project",
+    ""
```
Comment (nicolasstucki) [inline compiler/src/dotty/tools/dotc/config/ScalaSettings.scala]: Why is it empty?

Comment (felixmulder) [inline compiler/src/dotty/tools/dotc/config/ScalaSettings.scala]: If the version isn't specified, it falls back to an empty string. This renders nicely on the site.

Comment (nicolasstucki) [inline compiler/src/dotty/tools/dotc/config/ScalaSettings.scala]: Ok

**Comment by nicolasstucki** [comment]
The new side bar looks nice :)

**Pull request merged**